packages/deploy-tooling/src/cli/index.ts

```diff
@@ -0,0 +1,134 @@
+import { Option, Command } from 'commander';
+import { ToolsLogger, ConsoleTransport, LogLevel } from '@sap-ux/logger';
+import { deploy, getConfigForLogging, replaceEnvVariables, undeploy, validateConfig } from '../base';
+import type { CliOptions, AbapDeployConfig } from '../types';
+import { NAME } from '../types';
+import { getArchive } from './archive';
+import { getDeploymentConfig, mergeConfig } from './config';
+
+/**
+ * Create an instance of a command runner for deployment.
+ *
+ * @param name - command name
+ * @returns instance of the command
+ */
+export function createCommand(name: 'deploy' | 'undeploy'): Command {
```

Comment:
Yes, the `uaa-*` parameters are not available here because they expose potentially sensitive information, similar to a username or password parameter. The rest of the parameters should be covered but maybe renamed e.g. `failfast` is `no-retry`. 
Or did we miss anything?

Reply:
How are we supporting uaa-* parameters for CI/CD deploy usage?

Reply:
We didn't but now we do with https://github.com/SAP/open-ux-tools/pull/690/commits/64be74c9139667bdc79e15c7e86f066a5e807bc4.